<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="4" passed="1" failed="2" skipped="1">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-05-04T08:36:50 IST" name="Default suite" finished-at="2022-05-04T08:36:58 IST" duration-ms="7950">
    <groups>
    </groups>
    <test started-at="2022-05-04T08:36:50 IST" name="Default test" finished-at="2022-05-04T08:36:58 IST" duration-ms="7950">
      <class name="Test.ZerodhaLoginTest">
        <test-method is-config="true" signature="configureExtentReport()[pri:0, instance:Test.ZerodhaLoginTest@21282ed8]" started-at="2022-05-04T08:36:50 IST" name="configureExtentReport" finished-at="2022-05-04T08:36:50 IST" duration-ms="191" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- configureExtentReport -->
        <test-method is-config="true" signature="LaunchBrowser()[pri:0, instance:Test.ZerodhaLoginTest@21282ed8]" started-at="2022-05-04T08:36:50 IST" name="LaunchBrowser" finished-at="2022-05-04T08:36:52 IST" duration-ms="2427" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LaunchBrowser -->
        <test-method signature="LoginWithValidCredentials()[pri:0, instance:Test.ZerodhaLoginTest@21282ed8]" started-at="2022-05-04T08:36:52 IST" name="LoginWithValidCredentials" finished-at="2022-05-04T08:36:53 IST" duration-ms="870" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LoginWithValidCredentials -->
        <test-method is-config="true" signature="addTestResult(org.testng.ITestResult)[pri:0, instance:Test.ZerodhaLoginTest@21282ed8]" started-at="2022-05-04T08:36:53 IST" name="addTestResult" finished-at="2022-05-04T08:36:53 IST" duration-ms="7" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=LoginWithValidCredentials status=SUCCESS method=ZerodhaLoginTest.LoginWithValidCredentials()[pri:0, instance:Test.ZerodhaLoginTest@21282ed8] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- addTestResult -->
        <test-method is-config="true" signature="LaunchBrowser()[pri:0, instance:Test.ZerodhaLoginTest@21282ed8]" started-at="2022-05-04T08:36:53 IST" name="LaunchBrowser" finished-at="2022-05-04T08:36:54 IST" duration-ms="1259" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LaunchBrowser -->
        <test-method signature="clickOnSigup()[pri:0, instance:Test.ZerodhaLoginTest@21282ed8]" started-at="2022-05-04T08:36:54 IST" name="clickOnSigup" finished-at="2022-05-04T08:36:54 IST" duration-ms="65" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at org.testng.Assert.assertTrue(Assert.java:55)
at Test.ZerodhaLoginTest.clickOnSigup(ZerodhaLoginTest.java:62)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- clickOnSigup -->
        <test-method is-config="true" signature="LaunchBrowser()[pri:0, instance:Test.ZerodhaLoginTest@21282ed8]" started-at="2022-05-04T08:36:55 IST" name="LaunchBrowser" finished-at="2022-05-04T08:36:56 IST" duration-ms="1353" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LaunchBrowser -->
        <test-method is-config="true" signature="addTestResult(org.testng.ITestResult)[pri:0, instance:Test.ZerodhaLoginTest@21282ed8]" started-at="2022-05-04T08:36:55 IST" name="addTestResult" finished-at="2022-05-04T08:36:55 IST" duration-ms="0" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=clickOnSigup status=FAILURE method=ZerodhaLoginTest.clickOnSigup()[pri:0, instance:Test.ZerodhaLoginTest@21282ed8] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- addTestResult -->
        <test-method signature="verifyLoginText()[pri:0, instance:Test.ZerodhaLoginTest@21282ed8]" started-at="2022-05-04T08:36:56 IST" name="verifyLoginText" finished-at="2022-05-04T08:36:57 IST" duration-ms="1099" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[The following asserts failed:
expected [Login Kite] but found [Login to Kite]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [Login Kite] but found [Login to Kite]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
at Test.ZerodhaLoginTest.verifyLoginText(ZerodhaLoginTest.java:92)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyLoginText -->
        <test-method is-config="true" signature="addTestResult(org.testng.ITestResult)[pri:0, instance:Test.ZerodhaLoginTest@21282ed8]" started-at="2022-05-04T08:36:57 IST" name="addTestResult" finished-at="2022-05-04T08:36:57 IST" duration-ms="0" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=verifyLoginText status=FAILURE method=ZerodhaLoginTest.verifyLoginText()[pri:0, instance:Test.ZerodhaLoginTest@21282ed8] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- addTestResult -->
        <test-method signature="clickOnForgotPassword()[pri:0, instance:Test.ZerodhaLoginTest@21282ed8]" started-at="2022-05-04T08:36:57 IST" name="clickOnForgotPassword" finished-at="2022-05-04T08:36:57 IST" depends-on-methods="Test.ZerodhaLoginTest.clickOnSigup" duration-ms="0" status="SKIP">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method ZerodhaLoginTest.clickOnForgotPassword()[pri:0, instance:Test.ZerodhaLoginTest@21282ed8] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method ZerodhaLoginTest.clickOnForgotPassword()[pri:0, instance:Test.ZerodhaLoginTest@21282ed8] depends on not successfully finished methods
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:102)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- clickOnForgotPassword -->
        <test-method is-config="true" signature="flushResults()[pri:0, instance:Test.ZerodhaLoginTest@21282ed8]" started-at="2022-05-04T08:36:57 IST" name="flushResults" finished-at="2022-05-04T08:36:58 IST" duration-ms="272" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- flushResults -->
      </class> <!-- Test.ZerodhaLoginTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
